config/example/config.toml

```diff
@@ -653,6 +653,15 @@ enable_space_ruins = false
 minimum_zlevels = 2
 # Maximum number of extra zlevels to generate and fill with ruins
 maximum_zlevels = 4
+# Minimum space ruin budget.
+space_ruin_budget_min = 750
+# Maximum space ruin budget.
+space_ruin_budget_max = 1000
+# Minimum lavaland ruin budget.
+lavaland_ruin_budget_min = 175
+# Maximum lavaland ruin budget.
+lavaland_ruin_budget_max = 325
+
 # List of all space ruins that can generate in the world
```

Comment:
```suggestion
lavaland_ruin_budget_max = 325
# List of all space ruins that can generate in the world
```

This is on me.

I'll do the production cfg without this, just accept when ya can.